Assets/XR/Settings/OpenXR Package Settings.asset

```diff
@@ -531,6 +531,26 @@ MonoBehaviour:
   company: Joe M
   priority: 0
   required: 0
+--- !u!114 &365089772130919049
```

Comment:
Do not commit this file, it will cause issues with OpenXR Hand Tracking and OpenXR controllers if changed.